submissions/nilamop/popup/index.html

```diff
@@ -0,0 +1,154 @@
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/maps.png" alt="maps">
+                                        <span>Maps</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/files.png" alt="files">
+                                        <span>Files</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/contacts.png" alt="contacts">
+                                        <span>Contacts</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/tanslation.png" alt="tanslation">
+                                        <span>Tanslation</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/videos.png" alt="videos">
+                                        <span>Videos</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/search.png" alt="search">
+                                        <span>Search</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/photos.png" alt="photos">
+                                        <span>Photos</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/apps.png" alt="apps">
+                                        <span>Apps</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/documents.png" alt="documents">
```

Comment:
For decorative images implemented with the `img` tag `alt` attribute should be an empty value. The icon is a decorative image.

Reply:
Still not fixed :cry: